src/components/dialog/DialogDismissibleView.tsx

```diff
@@ -65,6 +63,8 @@ const DialogDismissibleView = (props: Props) => {
 
   const width = useRef<number>(Constants.screenWidth);
   const height = useRef<number>(Constants.screenHeight);
+  const TOP_INSET = useRef<number>(Constants.isIphoneX ? Constants.getSafeAreaInsets().top : Constants.isIOS ? 20 : 0);
+  const BOTTOM_INSET = useRef<number>(Constants.isIphoneX ? Constants.getSafeAreaInsets().bottom : Constants.isIOS ? 20 : 0);
```

Comment:
Looking at it now, it feels this kind of logic should be elsewhere, I guess in Constants. WDYT? 
I can approve and we can refactor later of course

Reply:
I'm adding a ticket for this, it'll require some adjusting in both public and private, so I'd rather not mix it in this fix.